index.Rmd

```diff
@@ -31,22 +31,20 @@ Raster data is like any image. Although it may portray various properties of obj
 
 Today's tutorial is about constructing a simple spatio-temporal analysis using raster data, R, and git.
 
-## Assumed knowledge from previous tutorials
-
-* Understand system architecture (R, R packages, libraries, drivers, bindings)
-* Good scripting habits
-
-
-# Reminder on overall system architecture
+# The overall system architecture
 
 
 ![R system architecture graph](figs/geoscripting-system-overview.svg){width=100%}
 
+The overall system architecture is comprised of integrated development environments (IDE), engines, packages, bindings, and libraries. Let's start with the engine, the core program that executes the foundation or crucial tasks of the programming language. The most relevant engines in this course are Python, Google Earth Engine, and R. To interact with the engine, we can either code in the command line directly or we can use an IDE, which provides many tools and features for working with the engine integrated in a single environment. An IDE allows the developer to write code, test it, and debug it all in a single software application. For interacting with the R engine, we can use *Rstudio* or *RKWard* for instance. 
+
```

Comment:
Since I don't think this has been covered previously, I have written a short description of the system architecture. Let me know what you think.

Reply:
Sounds good! The rest of the image shows some of the common packages, and how data gets stored on Git.